.prettierrc

```diff
@@ -0,0 +1,3 @@
+singleQuote: true
```

Comment:
@notrab Move Prettier configuration to `package.json`?

Reply:
@ynnoj I think in this case it's fine being in a dotfile since we have a load of others + all `moltin-examples` are now separated into their own dotfiles.